src/librustc_trans/context.rs

```diff
@@ -726,10 +725,6 @@ impl<'b, 'tcx> CrateContext<'b, 'tcx> {
         &self.shared.tcx.sess
     }
 
-    pub fn builder<'a>(&'a self) -> Builder<'a, 'tcx> {
-        Builder::new(self)
-    }
-
     pub fn raw_builder<'a>(&'a self) -> BuilderRef {
         self.local().builder.b
```

Comment:
Also `OwnedBuilder` should be the only `Builder`, i.e. the latter should not be a distinct type.